lakefile.lean

```diff
@@ -17,23 +20,3 @@ def runCmd (cmd : String) (args : Array String) : ScriptM Bool := do
   if hasError then
     IO.eprint out.stderr
   return hasError
-
-script build do
```

Comment:
Perhaps it's worth leaving `lake run build` in place and simply updating it -- that makes it slightly easier for someone to build the `md` files if needed, without needing to look up `mdgen`'s CLI syntax.